controller/transactions.go

```diff
@@ -99,6 +102,7 @@ func (t *Transactions) AllTemplatesForProjectID(ID uuid.UUID) ([]*model.Transact
 }
 
 func (t *Transactions) AllExecutionsForProjectID(ID uuid.UUID) ([]*model.TransactionExecution, error) {
```

Comment:
Don't keep commented out code if not needed. But I feel this should at least include get tx execution

Reply:
Removed the file for now